index.bs

```diff
@@ -14555,6 +14682,11 @@ expression syntax [[!PERLRE]]) as follows:
         <td><code>=</code></td>
         <td><code class="regex"><span class="mute">/</span>-?(([0-9]+\.[0-9]*|[0-9]*\.[0-9]+)([Ee][+-]?[0-9]+)?|[0-9]+[Ee][+-]?[0-9]+)<span class="mute">/</span></code></td>
     </tr>
+    <tr>
+        <td id="prod-bigint"><emu-t class="regex">bigint</emu-t></td>
+        <td><code>=</code></td>
+        <td><code class="regex"><span class="mute">/</span>-?([1-9][0-9]*|0[Xx][0-9A-Fa-f]+|0[0-7]*)n<span class="mute">/</span></code></td>
```

Comment:
The combined lexical goal can currently be realized case insensitively (assuming one handles unnamed terminals in a lookup step). That will cease to be true after this change because this "n" is case sensitive. This isn’t a problem, but I’m mentioning it because it could plausibly impact existing Web IDL lexers if they’ve made certain assumptions.

Reply:
I don't think we actually need a bigint token; integer works fine. I did that instead.